### PR TITLE
Connector broken links

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixing broken links and adding base path in connector docs + some minor fixes.

### CODE REVIEW

File: en/docs/reference/connectors/ISO8583-connector/ISO8583-connector-example.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 Follow these steps to set up the Integration Project and the Connector Exporter Project.
 
-{!references/connectors/importing-connector-to-integration-studio.md!}
+{!reference/connectors/importing-connector-to-integration-studio.md!}
```
Comment (nilminiwso2) [inline en/docs/reference/connectors/ISO8583-connector/ISO8583-connector-example.md:21]: Shall we remove the .md extensions from all the files?

Comment (samgnaniah) [reply en/docs/reference/connectors/ISO8583-connector/ISO8583-connector-example.md:21]: Fixed.

Comment (samgnaniah) [reply en/docs/reference/connectors/ISO8583-connector/ISO8583-connector-example.md:21]: Actually we can't remove it from this particular one. The links break without .md as these are files getting pulled into the page and it doesn't work without the extension.

Comment (nilminiwso2) [reply en/docs/reference/connectors/ISO8583-connector/ISO8583-connector-example.md:21]: ack

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**